tests/integration/components/lt-head-test.js

```diff
@@ -62,6 +63,35 @@ test('click - sortable column', function(assert) {
   click(sortableHeader);
 });
 
+test('render sort icons', function(assert) {
+  this.set('table', new Table(Columns));
+
+  this.render(hbs`{{lt-head table=table renderInPlace=true iconSortable='fa-sort' iconAscending='fa-sort-asc' iconDescending='fa-sort-desc'}}`);
+
+  let allHeaders = findAll('tr > th ');
+  let sortableHeader = allHeaders[allHeaders.length - 1];
+  let sortIcon = find('.lt-sort-icon', sortableHeader);
+
+  // Sortable case
+  assert.ok(hasClass(sortIcon, 'fa-sort'), 'Sortable icon renders');
```

Comment:
good test cases! I think it's fine as 1.

this also brought to light for me that the default behavior when sortable is enabled is to show the sortable icons.
If someone is expecting/want it not to show up it could be seen as a "breaking" change that the icon now show up.

Do we care? Should we document how turn it off? Have a cookbook example where it only shows on hover? Have a `showSortableIconByDefault` configuration option at the `ember-cli-build.js` level?